commit 2dbc752d644440f963bad172cf3ec83dc681ec7f
Author: Mauro Carvalho Chehab <mchehab@s-opensource.com>
Date:   Tue Mar 7 06:54:39 2017 -0300

    CMakeLists.txt: fix it to allow building from tarball
    
    Kaffeine has a script at tools/update_l10n.sh that allows one to
    download and test all translations locally, with is useful during
    Kaffeine development. Such script creates a file at po/CMakeLists.txt.
    
    However, normal users will use the Kaffeine tarball, downloaded from
    kde.org.
    
    The translations at the po/ directory on the tarballs don't
    have a CMakeLists.txt, as this file is created only if the
    tools/update_l10n.sh is called.
    
    So, change the main CMakeLists.txt to cope with both ways.
    
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3215e94..d24e848 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -93,7 +93,11 @@ add_subdirectory(icons)
 add_subdirectory(profiles)
 add_subdirectory(src)
 if (IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
-  add_subdirectory(po)
+  if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/po/CMakeLists.txt" )
+    add_subdirectory(po)
+  else()
+    ki18n_install(po)
+  endif()
 endif()
 
 if(KF5DocTools_VERSION)
